src/lando/ui/jobs.py

```diff
@@ -124,3 +125,8 @@ def get(
 class AutomationJobView(BaseJobView):
     job_type = AutomationJob
     worker_type = WorkerType.AUTOMATION
+
+
+class UpliftJobView(BaseJobView):
+    job_type = UpliftJob
+    worker_type = WorkerType.UPLIFT
```

Comment:
🎉

Reply:
Your refactor made setting up the new worker a breeze! :tada: